app-kotlin/src/main/res/layout/error_view.xml

```diff
@@ -1,11 +1,24 @@
 <?xml version="1.0" encoding="utf-8"?>
-<ImageView xmlns:android="http://schemas.android.com/apk/res/android"
+<FrameLayout xmlns:android="http://schemas.android.com/apk/res/android"
     xmlns:app="http://schemas.android.com/apk/res-auto"
-    android:id="@+id/image_error"
-    android:layout_width="100dp"
-    android:layout_height="100dp"
-    android:layout_gravity="center"
-    android:visibility="gone"
-    app:contentDescription="error image"
-    app:srcCompat="@drawable/ic_error_round" />
+    android:layout_width="match_parent"
+    android:layout_height="match_parent">
 
+    <ProgressBar
+        android:id="@+id/progressBar"
+        android:layout_width="20dp"
+        android:layout_height="20dp"
+        android:layout_gravity="center"
+        android:visibility="gone" />
```

Comment:
progress bar in a error view? 🤔 i feel `error_view` is a misnomer

Reply:
yes this is what i exactly think in recent pr to do thanks for mentioning i am fixing this in amend commit